src/ansible/roles/ipasmartcard/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+- block:
+  ################# IPA Server ####################
+  - name: Run IPA advise script for server setup for smart cards
+    shell: |
+      kinit admin@IPA.TEST
+      ipa-advise config-server-for-smart-card-auth > /tmp/sc_server.sh
+      sh -x /tmp/sc_server.sh {{ sssd_ca_path }} {{ ipa_ca_path }}
+    args:
+      stdin: '{{ ipa_password }}'
+    register: ipa_advise_sc_server
+
+  - name: Workaround to disable IPA WebUI OCSP checking
+    lineinfile:
+      path: /etc/httpd/conf.d/ssl.conf
+      regexp: 'SSLOCSPEnable on'
+      line: 'SSLOCSPEnable on no_ocsp_for_cert_ok'
+      backrefs: yes
+
+  - name: Restart httpd service
+    ansible.builtin.systemd_service:
+      name: httpd
+      state: restarted
+
+  when: inventory_hostname == 'master.ipa.test'
```

Comment:
I would target group ipa instead and make sure that it uses variables where appropriate (like kinit).

Reply:
That was one of the things I wanted input on.   Take a look at the update and let me know if I missed anything or if there's a better (or more standard to the project) way to handle this.